src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -360,22 +367,22 @@ def shim_to_feedback(
                 feedback_event["contexts"]["trace"] = event.data["contexts"]["trace"]
 
             feedback_event["timestamp"] = event.datetime.timestamp()
-            feedback_event["level"] = event.data["level"]
             feedback_event["platform"] = event.platform
             feedback_event["level"] = event.data["level"]
             feedback_event["environment"] = event.get_environment().name
-            feedback_event["tags"] = [list(item) for item in event.tags]
 
-        else:
+            tags = {}
+            for [k, v] in event.tags:
```

Comment:
Should check if event.tags is a list or a dict. I think the two formats are
[[k, v], [k, v]]
and 
{k:v, k:v}

Reply:
Actually I don't think we need this conversion here, just in `fix_for_issue_platform`.

Reply:
i think we only need to convert the tags from list to dict in the shim part, i think tags are always dicts in the fix_for_issue_platform

Reply:
Here: https://github.com/getsentry/sentry/blob/2b87f2a655b22fd8a22722b13905f195183264dc/src/sentry/feedback/usecases/create_feedback.py#L253-L253

Reply:
We don't have a guarantee the event passed to `fix_for_issue_platform` has a tag dict, or a tag list. So I think it'd definitely help to force it into a dict in that fx. In `shim_to_feedback` it's kinda optional (can tweak some tests if needed).

Reply:
if we don't know if it's even a list, how can we turn it into a dict?

Reply:
You can use `isinstance(tags, list)`

Reply:
my question is, if a string or something garbage is passed in how are we supposed to turn that into a dict?

Reply:
or are you saying the two possible types are _only_ list or dict?

Reply:
yeah, those are the 2 possibilities

Reply:
okay it should be modified!